_data/chains/eip155-314159.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Filecoin — Calibration testnet",
+  "chain": "FIL",
+  "status": "incubating",
+  "rpc": [],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "tFIL",
+    "symbol": "tFIL",
+    "decimals": 18
+  },
+  "infoURL": "https://filecoin.io",
+  "shortName": "filecoin",
+  "icon": "filecoin",
+  "chainId": 314159,
+  "networkId": 314159,
+  "slip44": 461,
```

Comment:
slip44 for Filecoin testnets is "1"

Reply:
Fixed in [`4fa8ff0` (#1567)](https://github.com/ethereum-lists/chains/pull/1567/commits/4fa8ff0d90481cc277f8b98859dbbe32e571ed33).